src/Impostor.Hazel/Udp/UdpConnection.Reliable.cs

```diff
@@ -379,12 +378,15 @@ private async ValueTask<bool> ProcessReliableReceive(ReadOnlyMemory<byte> bytes,
                     //See if we're missing it, else this packet is a duplicate as so we return false
                     if (!reliableDataPacketsMissing.Remove(id))
                     {
-                        return false;
+                        result = false;
                     }
                 }
             }
+            
+            //Send an acknowledgement
+            await SendAck(id);
 
-            return true;
+            return result;
```

Comment:
Can you link the PR of the changes related to this change? What does the change accomplish?

Reply:
Seems like it was in reactor repo for some time, pretty sure someone dm'ed me with this https://github.com/willardf/Hazel-Networking/pull/8